src/core/agent/services/identifierService.test.ts

```diff
@@ -410,42 +407,42 @@ describe("Single sig service of agent", () => {
 
     PeerConnection.peerConnection.getConnectingIdentifier = jest
       .fn()
-      .mockReturnValue({ id: archivedMetadataRecord.id, oobi: "oobi" });
-    await identifierService.deleteIdentifier(archivedMetadataRecord.id);
+      .mockReturnValue({ id: identifierMetadataRecord.id, oobi: "oobi" });
+    await identifierService.deleteIdentifier(identifierMetadataRecord.id);
     expect(connections.deleteConnectionById).toBeCalledWith("group-id");
     expect(identifierStorage.updateIdentifierMetadata).toBeCalledWith(
-      archivedMetadataRecord.id,
+      identifierMetadataRecord.id,
       {
         isDeleted: true,
       }
     );
   });
 
-  test("can delete an archived identifier (identifier and metadata record)", async () => {
+  test("can delete an identifier (identifier and metadata record)", async () => {
```

Comment:
I think we can remove this test - "can delete an identifier and disconnect DApp" covers everything

Reply:
@Sotatek-TungNguyen2a You have renamed the test instead of removing it. Now we have 2 tests with the same name. Remove the first shorter one, and keep the one that actually tests "disconnects DApp"

Reply:
Sorry for not reading carefully

Reply:
no problem!